File: src/main/java/net/atos/zac/flowable/task/exception/TaskNotFoundException.java
/*
 * SPDX-FileCopyrightText: 2024 INFO.nl
 * SPDX-License-Identifier: EUPL-1.2+
 */

package net.atos.zac.flowable.task.exception;

import jakarta.ws.rs.NotFoundException;

public class TaskNotFoundException extends NotFoundException {
    public TaskNotFoundException(final String message) {
        super(message);
    }

    public TaskNotFoundException(final String message, final Throwable cause) {
        super(message, cause);
    }
}

File: src/main/kotlin/nl/info/zac/flowable/task/exception/TaskNotFoundException.kt
/*
 * SPDX-FileCopyrightText: 2024 INFO.nl
 * SPDX-License-Identifier: EUPL-1.2+
 */

package nl.info.zac.flowable.task.exception

import jakarta.ws.rs.NotFoundException

class TaskNotFoundException(message: String) : NotFoundException(message) {
    constructor(message: String, cause: Throwable) : this(message) {
        initCause(cause)
    }
}
